tests/cms/test_media_library.py

```diff
@@ -0,0 +1,465 @@
+from __future__ import annotations
+
+import os
+from urllib import parse
+
+import pytest
+from django.conf import settings
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test.client import Client
+from django.urls import reverse
+
+from integreat_cms.cms.models import Directory, MediaFile
+from tests.conftest import (
+    ANONYMOUS,
+    HIGH_PRIV_STAFF_ROLES,
+    PRIV_STAFF_ROLES,
+    STAFF_ROLES,
+)
+
+
+@pytest.mark.django_db
+def test_directory_path(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+) -> None:
+    # Log the user in
+    client, role = login_role_user
+
+    query = parse.urlencode(
+        {
+            "directory": 1,
+        }
+    )
+    next_url = reverse("mediacenter_directory_path")
+    url = f"{next_url}?{query}"
+
+    response = client.post(url)
```

Comment:
```suggestion
    response = client.get(url)
```

We handle both methods, but I think it's more important to test GET because the GET request is sent from JS.